dsgrid/config/supplemental_dimension.py

```diff
@@ -19,3 +20,23 @@ class SupplementalDimensionModel(DimensionModel):
             title="mapping",
         ),
     ]
+
+
+class SupplementalDimensionsListModel(DSGBaseModel):
+    """Defines a list of supplemental dimensions."""
+
+    supplemental_dimensions: Annotated[
+        list[SupplementalDimensionModel],
+        Field(
+            title="supplemental_dimensions",
+            description="List of supplemental dimensions. They will be automatically registered "
+            "during project registration and then converted to supplemental_dimension_references.",
```

Comment:
Is this model only used for the new CLI functionality to add supplemental dimensions to an already-registered project? If so, should this description say something different than "They will be automatically registered during project registration"?